.idea/misc.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is editor stuff. Should not be committed. Please remove.

Reply:
I am happy that this functionality was added to this package. I do keep contributing. The reason for those extra error catchers is explained below. I had to do it that way to test Fin-hub servers.